client/cmd/client.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"github.com/lidofinance/dc4bc/client/api"
+	"github.com/lidofinance/dc4bc/client/config"
+	"github.com/lidofinance/dc4bc/client/services"
+	"github.com/lidofinance/dc4bc/storage/kafka_storage"
+	"github.com/segmentio/kafka-go/sasl/plain"
+	"log"
+	"os"
+	"strings"
+)
+
+func main() {
+	conf := &config.Config{
```

Comment:
How does this work? Lots of hardcoded values, configuration seems to be missing.